query/src/catalogs/mod.rs

```diff
@@ -17,11 +17,9 @@ pub use catalog::Catalog;
 pub use database::Database;
 pub use impls::util::in_memory_metas::InMemoryMetas;
 pub use meta_id_ranges::*;
-//pub use meta_backend::MetaBackend;
```

Comment:
I guess this can be removed in the next patch

Reply:
OK, will be clean in PR #2026